python/GafferSceneUI/LightEditor.py

```diff
@@ -381,19 +379,52 @@ def __buttonPress( self, pathListing, event ) :
 
```

Comment:
> pathListing.setSelection( selection )

This is sort-of or a pre-existing problem, but it's worse now we're accepting clicks on the name column. We need to pass `expandNonLeaf = False` to `setSelection()`, otherwise a right click can trigger unwanted expansions. Perhaps we should pass `scrollToFirst = False` too?

Reply:
Fixed in 2c05467872d4612bb15926e93eb3f4c4c414b794